agent/xdsv2/rbac_resources.go

```diff
@@ -143,7 +144,11 @@ func makeRBACPolicies(l4Permissions []*pbproxystate.Permission) map[string]*envo
 	policies := make(map[string]*envoy_rbac_v3.Policy, len(l4Permissions))
 
 	for i, permission := range l4Permissions {
-		policy := makeRBACPolicy(permission)
+		if len(permission.DestinationRules) != 0 {
```

Comment:
Since this looks like a CVE to a casual reader of the code, It's probably worth adding a comment that ports are taken into account before the IR and don't exist on destination rules anymore.